tests/test_infer_ops/triton/test_layernorm_triton.py

```diff
@@ -22,22 +23,19 @@
 @parameterize("M", [2, 4, 8, 16])
 @parameterize("N", [64, 128])
 def test_layer_norm(M, N):
+
     dtype = torch.float16
     eps = 1e-5
     x_shape = (M, N)
     w_shape = (x_shape[-1],)
-    weight = torch.rand(w_shape, dtype=dtype, device="cuda")
-    bias = torch.rand(w_shape, dtype=dtype, device="cuda")
+    weight = torch.ones(w_shape, dtype=dtype, device="cuda")
+    rms_norm = LlamaRMSNorm(hidden_size=N, eps=eps).cuda()
     x = -2.3 + 0.5 * torch.randn(x_shape, dtype=dtype, device="cuda")
 
-    y_triton = layer_norm(x, weight, bias, eps)
-    y_torch = torch.nn.functional.layer_norm(x, w_shape, weight, bias, eps).to(dtype)
-
-    assert y_triton.shape == y_torch.shape
-    assert y_triton.dtype == y_torch.dtype
-    print("max delta: ", torch.max(torch.abs(y_triton - y_torch)))
-    assert torch.allclose(y_triton, y_torch, atol=1e-2, rtol=0)
+    y_triton = layer_norm(x, weight, eps=eps)
+    y_llama = rms_norm.forward(x).to(dtype)
 
+    assert torch.allclose(y_triton, y_llama, atol=1e-5, rtol=1e-5)
 
```

Comment:
A benchmark comparing triton version and torch version is expected to be added here.